src/main/schemas/projectoraspectratios.schema.json

```diff
@@ -0,0 +1,87 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "$id": "http://isdcf.com/ns/json-schemas/registries/projectoraspectratios/1.0.0-beta.1",
+  "$comment": "Copyright, ISDCF <info@isdcf.com>",
+  "title": "Schema for the Projector Aspect Ratios and Resolutions Registry of the ISDCF",
+  "definitions": {
+    "dcncCode" : {
+        "type": "string",
+        "pattern": "^\\[?[A-Z0-9]\\]?"
+    },
+    "cplMetadata": {
+      "type": "object",
+      "properties": {
+        "definingDoc": {
+          "type": "array"
+        },
+        "2K": {
+          "$ref": "#/definitions/2K"
+        },
+        "4K": {
+          "$ref": "#/definitions/4K"
+        }
+      },
+      "required": [ "definingDoc", "2K", "4K"]
+    },
+    "2K": {
+      "type": "object",
+      "properties": {
+        "ScreenAspectRatio": {
+          "type": "string"
```

Comment:
```suggestion
          "type": "string",
          "pattern": "^\\d+ \\d+?"
```

Reply:
Oh, good one! I also added in an item on the test for this as well.

Reply:
Addressed in f5a6107